telos-frontend/src/components/todo/DashboardTodo.jsx

```diff
@@ -23,6 +24,11 @@ import MoreVertIcon from '@material-ui/icons/MoreVert';
 import ErrorIcon from '@material-ui/icons/Error';
 import Menu from '@material-ui/core/Menu';
 import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
```

Comment:
Could you import like:
import { Button, Dialog, etc } from '@material-ui/core'?